resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosClusterScheduler.scala

```diff
@@ -374,6 +389,16 @@ private[spark] class MesosClusterScheduler(
     s"${frameworkId}-${desc.submissionId}${retries}"
   }
 
+  private[mesos] def getDriverPriority(desc: MesosDriverDescription): Float = {
+    val queueName =
+      desc.conf.get("spark.mesos.dispatcher.queue", config.DISPATCHER_QUEUE.defaultValueString)
+    if (queueName != config.DISPATCHER_QUEUE.defaultValueString) {
+      queues.getOrElse(queueName, throw new NoSuchElementException(queueName))
```

Comment:
Do we have a test case for this `NoSuchElementException`?

Reply:
:+1: